tests/foreman/ui/test_activationkey.py

```diff
@@ -50,6 +50,8 @@ def setUpClass(cls):
         cls.org_label = org_attrs['label']
         cls.org_name = org_attrs['name']
         cls.org_id = org_attrs['id']
+        cls.base_key_name = entities.ActivationKey(
+            organization=cls.org_id).create()['name']
```

Comment:
Thank you. This is a subtle thing that has no ill effect right now, so it's easy to miss. :)